sycl/plugins/unified_runtime/ur/adapters/hip/context.hpp

```diff
@@ -27,7 +27,7 @@ typedef void (*ur_context_extended_deleter_t)(void *UserData);
 /// UR API context are objects that are passed to functions, and not bound
 /// to threads.
 /// The ur_context_handle_t_ object doesn't implement this behavior. It only
-/// holds the HIP context data. The RAII object \ref ScopedContext implements
+/// holds the HIP context data. The RAII object \ref ScopedDevice implements
 /// the active context behavior.
```

Comment:
Would it be worth moving this to `device.hpp` and changing the comment a bit?

Reply:
done